packages/vitest/src/node/reporters/renderers/windowedRenderer.ts

```diff
@@ -77,6 +78,20 @@ export class WindowRenderer {
     clearInterval(this.renderInterval)
   }
 
+  /**
+   * Queue new render update
+   */
+  schedule() {
+    if (!this.renderScheduled) {
+      this.renderScheduled = true
+      this.flushBuffer()
+
+      setTimeout(() => {
+        this.renderScheduled = false
+      }, 100).unref()
+    }
+  }
+
```

Comment:
This is now similar to Jest: https://github.com/jestjs/jest/blob/5f26f09cc5bbf0d82faa6f91f77283db7246cfbb/packages/jest-reporters/src/Status.ts#L210-L220

And for `DEFAULT_RENDER_INTERVAL_MS`: https://github.com/jestjs/jest/blob/5f26f09cc5bbf0d82faa6f91f77283db7246cfbb/packages/jest-reporters/src/Status.ts#L108